app/mm-redux/actions/channels.ts

```diff
@@ -1517,6 +1517,14 @@ export function getChannelMemberCountsByGroup(channelId: string, includeTimezone
     };
 }
 
+export function isChannelMember(channelId: string): ActionFunc {
+    return (_dispatch: DispatchFunc, getState: GetStateFunc) => {
+        return {
+            data: Boolean(getState().entities.channels?.membersInChannel[channelId]?.channel_id),
```

Comment:
Maybe we can do this instead (I just don't recall seeing `Boolean` used)
```suggestion
            data: getState().entities.channels?.membersInChannel[channelId]?.channel_id || false,
```

Reply:
is fine to use `Boolean`, something you just learned @hahmadia

Reply:
I do agree this should be either an utility function or a selector, and utility function makes more sense as you don't really need to memoize it.

Reply:
cool thanks

Reply:
Thanks! I have moved it to utils.